config/types/schema.go

```diff
@@ -151,7 +149,7 @@ type PasswdGroup struct {
 type PasswdUser struct {
 	Create            *Usercreate        `json:"create,omitempty"`
 	Gecos             string             `json:"gecos,omitempty"`
-	Groups            []Group            `json:"groups,omitempty"`
```

Comment:
Ok, as long as schematyper is generating it. From looking at the schema, I'm not sure how it decided to do that, but as long as it consistently produces that :+1: